src/parser/classes/ReelPlayerOverlay.ts

```diff
@@ -0,0 +1,40 @@
+import Parser, { type RawNode } from '../index.js';
+import {YTNode} from '../helpers.js';
+import Button from './Button.js';
+import Menu from './menus/Menu.js';
+import InfoPanelContainer from './InfoPanelContainer.js';
+import LikeButton from './LikeButton.js';
+import ReelPlayerHeader from './ReelPlayerHeader.js';
+import PivotButton from './PivotButton.js';
+import Text from './misc/Text.js';
+
+export default class ReelPlayerOverlay extends YTNode {
+  static type = 'ReelPlayerOverlay';
+
+  like_button: LikeButton | null;
+  reel_player_header_supported_renderers: ReelPlayerHeader | null;
+  menu: Menu | null;
+  next_item_button: Button | null;
+  prev_item_button: Button | null;
+  subscribe_button_renderer: Button | null;
+  style: string;
+  view_comments_button: Button | null;
+  share_button: Button | null;
+  pivot_button: PivotButton | null;
+  info_panel: InfoPanelContainer | null;
+
+  constructor(data: RawNode) {
+    super();
+    this.like_button = Parser.parseItem(data.likeButton, [ LikeButton ]);
```

Comment:
When you are just passing one class to parse, you don't need the array around it, you can just pass it directly.
```suggestion
    this.like_button = Parser.parseItem(data.likeButton, LikeButton);
```

Reply:
Done :D